Src/BootCamp.Chapter/Account.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    public class Account
+    {
+        private readonly string _name;
+        private readonly decimal[] _balance;
+
+        public Account(string balance)
+        {
+            _name = ArrayOps.GetNameForPerson(balance);
+            _balance = ArrayOps.GetBalanceForPerson(balance);
+        }
+
+        public string GetName()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Consider making method 'GetName' a property.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5054323)